jdaviz/core/helpers.py

```diff
@@ -273,96 +274,96 @@ def get_model_parameters(self, models=None, model_label=None, x=None, y=None):
+
+            "inline": Display the Jdaviz application inline in a notebook.
+            Note this is functionally equivalent to displaying the cell
+            ``viz.app`` in the notebook.
+
+            "sidecar": Display the Jdaviz application in a separate JupyterLab window from the
+            notebook, the location of which is decided by the 'anchor.' right is the default
+
+                Other anchors:
+                sidecar:right        | The default, opens a tab to the right of display
+                sidecar:tab-before   | Full-width tab before the current notebook
+                sidecar:tab-after    | Full-width tab after the current notebook
+                sidecar:split-right  | Split-tab in the same window right of the notebook
+                sidecar:split-left   | Split-tab in the same window left of the notebook
+                sidecar:split-top    | Split-tab in the same window above the notebook
+                sidecar:split-bottom | Split-tab in the same window below the notebook
+
+                See `jupyterlab-sidecar <https://github.com/jupyter-widgets/jupyterlab-sidecar>`_
+                for the most up-to-date options.
+
+
         title : str, optional
             The title of the sidecar tab.  Defaults to the name of the
             application; e.g., "specviz".
-        anchor : str
-            Where the tab should appear, by default on the right. Options are:
-            {sidecar_anchor_values}.
+
+            NOTE: Only applicable to a "sidecar" display.
 
         Returns
         -------
-        sidecar
-            The ``sidecar.Sidecar`` object used to create the tab.
+        sidecar or None
+            The ``sidecar.Sidecar`` object used to create the tab or window or
+            None if no sidecar was created
 
         Notes
         -----
-        If this method is called in the "classic" Jupyter notebook, the app will
-        appear inline, as only lab has a mechanism to have multiple tabs.
-        See Also
-        --------
-        show_in_new_tab
-        show_inline
+        If "sidecar" is requested in the "classic" Jupyter notebook, the app will appear inline,
+        as only lab has a mechanism to have multiple tabs.
         """
-        if 'title' not in kwargs:
-            kwargs['title'] = self.app.config
+        try:
+            if loc == "inline":
+                display(self.app)
 
-        scar = Sidecar(**kwargs)
-        with scar:
-            display(self.app)
+            elif loc.startswith('sidecar'):
+                if loc != 'sidecar':
+                    anchor = loc.split(':')[1]
+                    kwargs['anchor'] = anchor
 
-        return scar
-    show_in_sidecar.__doc__ = show_in_sidecar.__doc__.format(
-        sidecar_anchor_values=repr(Sidecar.anchor.values)[1:-1])
+                # If title unset, default to the viz config
+                if 'title' not in kwargs:
+                    kwargs['title'] = self.app.config
```

Comment:
This should be a little more efficient, but effectively does the same thing:

```suggestion
                kwargs.setdefault('title', self.app.config)
```

(but might be irrelevant depending on what you decide on my comment about `**kwargs`)

Reply:
Dropping **kwargs, so this is irrelevant as pointed out above